view/multi_match.js

```diff
@@ -0,0 +1,43 @@
+/**
+  this view is wrapped in a function so it can be re-used
+**/
+
+module.exports = function( properties ){
+  return function( vs ){
+
+    // base view
+    var view = { multi_match: {} };
+
+    properties.forEach(function(property) {
```

Comment:
what's the intent of this `forEach`? it seems to be doing nothing. was it meant to be a `filter`?